streams/src/main/java/org/apache/kafka/streams/state/WindowStoreUtils.java

```diff
@@ -19,13 +19,18 @@
 
 package org.apache.kafka.streams.state;
 
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serdes;
+
 import java.nio.ByteBuffer;
 
 public class WindowStoreUtils {
 
-    public static final int TIMESTAMP_SIZE = 8;
     public static final int SEQNUM_SIZE = 4;
-    public static final StateSerdes<byte[], byte[]> INNER_SERDES = StateSerdes.withBuiltinTypes("", byte[].class, byte[].class);
+    public static final int TIMESTAMP_SIZE = 8;
```

Comment:
Are these `public static final` fields like `TIMESTAMP_SIZE`, `SEQNUM_SIZE`, `INNER_SERDE` really part of the public API and its contract?  If they really are, we should add a docstring to each of these.